.github/workflows/code_analysis.yaml

```
@@ -54,6 +54,7 @@ jobs:
                     php-version: ${{ matrix.php-version }}
                     extensions: json, mbstring, pdo, curl, pdo_sqlite
                     coverage: none
+                    tools: composer, symfony-cli
```

Maybe we should specify the major here to prevent surprises down the line?

```suggestion
                    tools: composer:v2, symfony-cli:5
```

I do see that the documentation of the actions mentions:

> With the exception of major versions of composer, if you specify only the major version or the version in major.minor format for a tool you can get rate limited by GitHub's API. To avoid this, it is recommended to provide a [GitHub OAuth token](https://github.com/shivammathur/setup-php#composer-github-oauth).

I cannot access the repository secrets just yet, but I'm still tempted to add this specification already. WDYT?

We will want to update this at all usages of the `shivammathur/setup-php@v2` task.